source/test/TestDirector.cc

```diff
@@ -227,6 +226,16 @@ bool TestDirector::sync() const {
     return m_sync;
 }
 
+void TestDirector::OnInit(System::RaceConfig *config, void * /* arg */) {
```

Comment:
What's the intent of `arg`? Should probably be a doxygen comment here to explain

Reply:
The intent of arg is explained in the brief where it is defined, in RaceConfig. Perhaps the actual callback brief should reference the arg.